docs/release-notes.txt

```diff
@@ -6,6 +6,47 @@ Release Notes
 Version 0.12
 ------------
 
+Version 0.12.13
+^^^^^^^^^^^^^^^
+
+**Release Date:** August 6, 2020
+
+**New Features**
+
+- Add the Determined Model Registry. The Model Registry makes it easy to organize trained models by providing versioning and labeling tools. See :ref:`organizing-models` to get started.
+
+- Add a new PyTorch interface to customize your workflows. See the :ref:`migration guide <migration-guide-flexible-primitives>` to upgrade your PyTorch interface. *Deprecated methods will be retained until at least the next minor release.*
+
+  - Support multiple models, optimizers, and LR schedulers. Instantiate those objects and wrap them with :meth:`PyTorchTrialContext.wrap_model`, :meth:`PyTorchTrialContext.wrap_optimizer`, and :meth:`PyTorchTrialContext.wrap_lr_scheduler` in :meth:`PyTorchTrial.__init__`. ``build_model``, ``optimizer``, and ``create_lr_scheduler`` in :class:`PyTorchTrial` are now deprecated.
+
+  - Support customizing forward and backward passes in :meth:`PyTorchTrial.train_batch`. Support gradient clipping by passing a function to the ``clip_grads`` argument of :meth:`PyTorchTrialContext.step_optimizer`. The callback ``on_before_optimizer_step`` is now deprecated.
+
+  - Configure automatic mixed precision in :meth:`PyTorchTrial.__init__`. The ``optimizations.mixed_precision`` experiment configuration key is now deprecated.
+
+  - The ``model`` arguments in :meth:`PyTorchTrial.train_batch`, :meth:`PyTorchTrial.evaluate_batch`, and :meth:`PyTorchTrial.evaluate_full_dataset` are now deprecated.
+
+- Implement :ref:`topic-guides_hp-tuning-det_adaptive-asha` for large-scale hyperparameter search. See the `associated paper <https://arxiv.org/pdf/1810.05934.pdf>`__ for more information.
+
+**Improvements**
+
+- Allow proxy environment variables to be set in the agent config. See :ref:`environment-variables` for more information.
+
+- Preserve random state for PyTorch experiments when checkpointing and restoring.
+
+- Remove ``det.pytorch.reset_parameters()``. This should have no effect except when using highly-customized ``nn.Module`` implementations.
```

Comment:
```suggestion
- Remove ``det.pytorch.reset_parameters()``. This should have no effect except when using highly customized ``nn.Module`` implementations.
```
(Professional style guides typically omit hyphens after "-ly" adverbs, since there's no ambiguity even without it.)